### PR TITLE
react/week3 homework

### PR DESCRIPTION
**Pull request by Saidemm**

### CODE REVIEW

File: react/week3/todo-list/src/components/Todo.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
```
Comment (nuhana) [inline react/week3/todo-list/src/components/Todo.js:1]: Try to choose more descriptive names for these files

File: react/week3/todo-list/src/components/Todos.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import moment from 'moment';
+
+
+
+export default function Todos({ todos, setTodos }) {
+    console.log('todos', todos);
```
Comment (nuhana) [inline react/week3/todo-list/src/components/Todos.js]: delete this

File: react/week3/todo-list/src/components/Todos.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    const [inputValue, setInputValue] = useState("");
+    const [deadline, setDeadline] = useState(null);
+
+    function getRandomTodo() {
+        let randomIndex = Math.floor(Math.random() * todos.length);
+        console.log(randomIndex);
+        console.log('todos', todos);
+        return todos[randomIndex];
```
Comment (nuhana) [inline react/week3/todo-list/src/components/Todos.js:19]: delete all console in your code

File: react/week3/todo-list/src/components/Todo.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        return (
+            setIsEditState(!isEditState)
+        )
+    }
+
+
+    const onUpdate = (id) => {     
+        const newTodos = todos.map(todo => {
+            if(todo.id === id){
+                todo.description = newDescription;
```
Comment (nuhana) [inline react/week3/todo-list/src/components/Todo.js]: return this also' ...todo' so the new array will be with the all todos 
may be like this` {...todo,todo.description = newDescription}`

**Review by nuhana — COMMENTED** [review]
Good job 👏  only few comments

File: react/week3/todo-list/src/components/Todo.js (hunk 1 above)
Comment (Saidemm) [reply react/week3/todo-list/src/components/Todo.js:1]: Yes, you are right. Thanks for your feedback.

File: react/week3/todo-list/src/components/Todo.js (hunk 4 above)
Comment (Saidemm) [reply react/week3/todo-list/src/components/Todo.js]: Thanks. i learned a new thing. Done.

**Pull request merged**